app/models/person.rb

```diff
@@ -28,6 +34,14 @@ class Person < ApplicationRecord
   #          -> { order created_at: :desc },
   #          class_name: 'ProtocolSubscription', foreign_key: 'filling_out_for_id'
 
+  after_initialize do |person|
+    next if person.id && person.external_identifier
```

Comment:
als person.id nil is, maar person.external_identifier is niet nil, dan overschrijf je nu person.external_identifier. Is dat wat je wilt? Of moet dit een || zijn ipv een && ?

Reply:
Nou, met een or gaat het niet goed, dan krijgt iemand nooit een external identifier als iemand al een person.id heeft. Het overschrijven klopt idd, maar is volgens mij niet een heel groot probleem.

Volgens mij is die check op person.id niet per se nodig, die komt uit een van onze andere classes.